wdl-analysis/tests/analysis/import-incompatible-versions/source.diagnostics

```diff
@@ -0,0 +1,12 @@
+error: unsupported WDL version `2.0`
+  ┌─ tests/analysis/import-incompatible-versions/foo.wdl:1:9
+  │
+1 │ version 2.0
+  │         ^^^ this version of WDL is not supported
+
+error: imported document has incompatible version `2.0`
```

Comment:
It would be a nice to have to put the original WDL version here as well to make it clear what version the import is being called from.